.github/workflows/benchmark.yml

```
@@ -13,7 +13,7 @@ env:
 jobs:
   benchmark:
     name: Benchmarks
-    runs-on: self-hosted
+    runs-on: github-benchmark-runner
```

TIL you can't target by runner name, only by labels, so I created a new label for this runner...
```suggestion
    runs-on: equinix-bare-metal
```